fastlane/Constants/Constant.swift

```diff
@@ -8,30 +8,37 @@
 
 enum Constant {
 
+    // MARK: - App Store
+
+    static let appStoreKeyId: String = "<#appStoreKeyId#>"
+    static let appStoreIssuerId: String = "<#appStoreIssuerId#>"
+
+    // MARK: - Firebase
+
+    static let stagingFirebaseAppId: String = "<#stagingFirebaseAppId#>"
+    static let productionFirebaseAppId: String = "<#productionFirebaseAppId#>"
+    static let firebaseTesterGroups: String = "<#group1#>, <#group2#>"
```

Comment:
Just curious, why don't we turn this into an array of Strings instead? 🤔
```suggestion
    static let firebaseTesterGroups: [String] = ["<#group1#>", "<#group2#>"]
```

Reply:
I think it's because in the final, it must convert to a string for Fastlane to send

Reply:
@minhnimble Good concern, you can check the [doc](https://firebase.google.com/docs/app-distribution/ios/distribute-fastlane) of firebase distribution. It is a string, so for me, I don't want extra logic to join them when passing to the function. What do you think?

![Screen Shot 2022-09-28 at 16 08 34](https://user-images.githubusercontent.com/17830319/192739218-3bd32f20-a82b-415b-ab46-3a1278bf7a27.png)

Reply:
noted, thanks for the sharing @vnntsu 👍